etc/kayobe/ansible/stackhpc-openstack-tests.yml

```diff
@@ -45,7 +46,7 @@
           ansible.builtin.pip:
             name:
               - "{{ repo_tmpdir.path }}"
-              - "-r{{ repo_tmpdir.path }}/requirements.txt"
+              - -r{{ repo_tmpdir.path }}/requirements.txt
```

Comment:
Why is there no space here? `-r{{` vs `-r {{`

Reply:
Not sure tbh, though that is how it originally was. I'll add a space because I agree, it's probably wrong

Reply:
Done